packages/browser-agent-core/src/common/deny-list/deny-list.ts

```diff
@@ -0,0 +1,117 @@
+import type { XhrParams } from "./deny-list.interfaces"
+
+/** An array of filter objects {hostname, pathname} for identifying XHR events to be excluded from collection.
+ * @see {@link https://docs.newrelic.com/docs/browser/new-relic-browser/configuration/filter-ajax-request-events/ Filter AjaxRequest events}
+ * @type {Array.<{hostname: string, pathname: string}>}
+ */
+ let denyList: Array<{ hostname: string; pathname: string }> = []
+
+ /**
+  * Evaluates whether an XHR event should be included for collection based on the {@link denyList|AjaxRequest deny list}.
+  * @param {{method: string, hostname: string, port: string, protocol: string, host: string, pathname: string, status: number}} params - object with properties of the XHR event
```

Comment:
[nit] With tsdoc you should be able to reference the interface instead of typing out the param type

Reply:
Pushed change. Good catch!